pkg/localstore/localstore.go

```diff
@@ -749,3 +788,27 @@ func totalTimeMetric(metric prometheus.Counter, start time.Time) {
 	metric.Add(float64(totalTime))
 
 }
+
+func isDirtyShutdown(path string) bool {
+	_, err := os.Stat(filepath.Join(path, sharkyDirtyFileName))
+	return !os.IsNotExist(err)
```

Comment:
This function predates `errors.Is`. It only supports errors returned by the os package. New code should use `errors.Is(err, fs.ErrNotExist)`.

Reply:
fixed